package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
I see this is a new file, is it needed?

Reply:
No. And it is also removed from my latest commits.